sound/soc/sof/intel/hda-dai.c

```diff
@@ -631,6 +625,12 @@ static int hda_dai_suspend(struct hdac_bus *bus)
 				if (ret < 0)
 					return ret;
 			}
+
+			ret = hda_link_dma_cleanup(hext_stream->link_substream,
+						   hext_stream,
+						   cpu_dai);
+			if (ret < 0)
+				return ret;
```

Comment:
I can't recall if this was an editing mistake or intentional. @ranj063 would you happen to remember this?

Reply:
@plbossart I dont think the order really matters in this case. All we do in the post trigger op for suspend is clear the count for pipeline.